index.html

```diff
@@ -448,33 +448,28 @@ <h3><a href="#blog-single.html">Business planning, strategy and execution</a> </
     <div class="top-footer">
       <div class="container my-md-5 my-4">
         <div class="row">
-          <div class="col-lg-4">
+          <div class="col-lg-2 ">
             <div class="footer-logo mb-3">
-              <a href="index.html">
-                <span class="fa fa-shield"></span> Captivate
-              </a>
-            </div>
-            <div>
-              <p class="">Lorem ipsum dolor, sit amet consectet et adipis icing elit. Ab commodi iure minus
-                laboriosam
-                placeat quia, dolorem animi. Eveniet repudiandae, perferendis nesciunt deserunt iure et, consequatur
-                optio!</p>
+              
+                <img class = "img-fluid" height="auto" src="assets/images/logos/acm/acm-color.svg" alt="ACM UWB logo"  >
```

Comment:
`img-fluid` already sets the height to auto. You shouldn't need to specify `height="auto"`